README.md

```diff
@@ -1,121 +1,115 @@
-This package builds on `scikit-learn`, `numpy` and `scipy` libraries.
-
-If you would like to contribute to `tslearn`, please have a look at [our contribution guidelines](CONTRIBUTING.md).
-
-# Dependencies
-
-```
-Cython
-numpy
-numba
-scipy
-scikit-learn
-joblib
-numba
-```
-
-If you plan to use the `shapelets` module, `keras` and `tensorflow` should also be installed.
-`h5py` is required for reading or writing models using the hdf5 file format.
+<!-- Our title -->
+<div align="center">
+  <h3>tslearn </h3>
+</div>
+
+<!-- Short description -->
+<p align="center"> 
+   "The machine learning toolkit for time series analysis in Python"
+</p>
+
+<!-- The badges -->
+<p align="center">
+    <a href="https://badge.fury.io/py/tslearn">
+        <img alt="PyPI" src="https://badge.fury.io/py/tslearn.svg">
+    </a>
+    <a href="http://tslearn.readthedocs.io/en/latest/?badge=latest">
+        <img alt="Documentation" src="https://readthedocs.org/projects/tslearn/badge/?version=latest">
+    </a>
+    <a href="https://travis-ci.org/rtavenar/tslearn">
+        <img alt="Build" src="https://travis-ci.org/rtavenar/tslearn.svg?branch=master">
+    </a>
+    <a href="https://codecov.io/gh/rtavenar/tslearn">
+        <img alt="Codecov" src="https://codecov.io/gh/rtavenar/tslearn/branch/master/graph/badge.svg">
+    </a>
+    <a href="https://pepy.tech/project/tslearn">
+        <img alt="Downloads" src="https://pepy.tech/badge/tslearn">
+    </a>
+</p>
+
+<!-- Draw horizontal rule -->
+<hr>
+
+<!-- Table of content -->
+
+| Section | Description |
+|-|-|
+| [Installing](#installation) | Installing the dependencies and tslearn |
+| [Getting started](#getting-started) | A quick introduction on how to use tslearn |
+| [Documentation](#documentation) | A link to our API reference and a gallery of examples |
+| [Contributing](#contributing) | A guide for heroes willing to contribute |
+| [Citation](#referencing-tslearn) | A citation for tslearn for scholarly articles |
 
 # Installation
-
-## Using conda
-
-The easiest way to install `tslearn` is probably via `conda`:
-```bash
-conda install -c conda-forge tslearn
+There are different alternatives to install tslearn:
+* PyPi: `python -m pip install tslearn`
+* Conda: `conda install -c conda-forge tslearn`
+* Git: `python -m pip install https://github.com/rtavenar/tslearn/archive/master.zip`
+
+In order for the installation to be successful, the required dependencies must be installed. For a more detailed guide on how to install tslearn, please see the [Documentation](https://tslearn.readthedocs.io/en/latest/?badge=latest#installation)
+
+# Getting started
+
+## 1. Getting the data in the right format
+tslearn expects the data to be 3D. The three dimensions correspond to the number of time series, the number of measurements per time series and the number of dimensions respectively (`n_ts, max_sz, d`). In order to get the data in the right format, different solutions exist:
```

Comment:
I would change to:

"tslearn expects a time series dataset to be formatted as a 3D `numpy` array".